outages/README.md

```diff
@@ -0,0 +1,55 @@
+## How to manage outages
+
+### Who is responsible for resolving outages and what should be done
+
+- rotating role - new / incorporate to an existing one (Sentry master, community master)
+- identify an outage
+- inform team
+- inform users
+- be responsible for fixing the issue = try to fix it / ask in the team
+- inform when the outage is resolved
+
+### How to identify an outage
+
+- Sentry
+  - mostly info about new exceptions
+  - currently we are informed about each new Sentry issue by mail
+  - [alerts](https://docs.sentry.io/product/alerts-notifications/metric-alerts/)
+    - alert rules can be set up - threshold, notification,
+      [example](https://www.sentry.dev/_assets2/static/fca0872133bf0ed54631d4ba44725879/eb1d2/new-alert-rule-all.png)
+    - alerting is triggered when a threshold is breached
+- Prometheus can provide alerts as well - we would need to collect some metric about failed tasks and create alert rules
+  for that
+- analyze DB info we use in dashboard jobs - periodically check the state of the SRPM builds/ Copr builds/ tests in DB from e.g. the last hour
+  - cron job to check this can be set up
+
+### How team members are informed
+
+- IRC - the messages are mirrored to Telegram, this should be enough
```

Comment:
Are you implying one of our current channels? AFAIK OSCI and CKI are routing their alerts to their main channels as well.